### PR TITLE
Implement loyalty panel and individual discounts

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Implement 'Individual discounts', 'Loyalty' panel](https://trello.com/c/suRV3Tf3/889-implement-individual-discounts-loyalty-panel)

File: lib/plugins.js [hunk 1]
```diff
@@ -252,10 +254,13 @@ function plugins (settings, deviceId) {
...
         const balances = arr.slice(cryptoCodesCount + 4, 2 * cryptoCodesCount + 4)
-        const testNets = arr.slice(2 * cryptoCodesCount + 4, arr.length - 1)
+        const testNets = arr.slice(2 * cryptoCodesCount + 4, arr.length - 2)
         const coinParams = _.zip(cryptoCodes, testNets)
         const coinsWithoutRate = _.map(mapCoinSettings, coinParams)
-        const areThereAvailablePromoCodes = arr[arr.length - 1] > 0
+        const areThereAvailablePromoCodes = arr[arr.length - 2] > 0
+        const individualDiscounts = arr[arr.length - 1]
+
+        console.log('individualDiscounts', individualDiscounts)
```
Comment (josepfo) [inline lib/plugins.js]: remove console log

**Review by josepfo — CHANGES_REQUESTED** [review]
Looking good! Good job!

**Review by josepfo — APPROVED** [review]

File: lib/plugins.js [hunk 2]
```diff
@@ -252,10 +254,11 @@ function plugins (settings, deviceId) {
         const cryptoCodesCount = cryptoCodes.length
         const tickers = arr.slice(4, cryptoCodesCount + 4)
         const balances = arr.slice(cryptoCodesCount + 4, 2 * cryptoCodesCount + 4)
-        const testNets = arr.slice(2 * cryptoCodesCount + 4, arr.length - 1)
+        const testNets = arr.slice(2 * cryptoCodesCount + 4, arr.length - 2)
         const coinParams = _.zip(cryptoCodes, testNets)
         const coinsWithoutRate = _.map(mapCoinSettings, coinParams)
-        const areThereAvailablePromoCodes = arr[arr.length - 1] > 0
+        const areThereAvailablePromoCodes = arr[arr.length - 2] > 0
+        const individualDiscounts = arr[arr.length - 1]
```
Comment (RafaelTaranto) [inline lib/plugins.js]: We should not send individualDiscounts for all customers, can't we just add it to the customer response?

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: lib/plugins.js (hunk 2 above)
Comment (chaotixkilla) [reply lib/plugins.js]: You're completely correct. Definitely an oversight on my part, I'll work on it

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**